docs/resources/Channel.md

```diff
@@ -1040,3 +1154,98 @@ Adds a recipient to a Group DM using their access token
+
+## Add a user to a thread % POST /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/{user.id#DOCS_RESOURCES_USER/user-object}
+
+Adds another user to a thread. Requires the ability to send messages in the thread.  Also requires the thread is not archived. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## Leave a thread % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/@me
+
+Removes the current user from a thread. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## Remove a user from a thread % DELETE /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/thread-members/{user.id#DOCS_RESOURCES_USER/user-object}
+
+Removes another user from a thread. Requires the `MANAGE_THREADS` permission or that you are the creator of the thread.  Also requires the thread is not archived. Returns a 204 empty response on success.  Fires a [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway event.
+
+## Get all public archived threads % GET /channels/{channel.id#DOCS_RESOURCES_CHANNEL/channel-object}/threads/archived/public
+
+Returns archived threads in the channel that are public.  When called on a `GUILD_TEXT` channel, returns threads of [type](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types) `PUBLIC_THREAD`.  When called on a `GUILD_NEWS` channel returns threads of [type](#DOCS_RESOURCES_CHANNEL/channel-object-channel-types) `NEWS_THREAD`.  Threads are ordered by `archive_timestamp`, in descending order. Requires the `READ_MESSAGE_HISTORY` permission.
+
+###### Query String Params
+
+| Field   | Type      | Description                                  |
+|---------|-----------|----------------------------------------------|
+| before? | timestamp | returns threads before this timestamp        |
+| limit?  | integer   | optional maximum number of threads to return |
+
+###### Response Body
+
+| Field    | Type      | Description                                                                                                                  |
+|----------|-----------|------------------------------------------------------------------------------------------------------------------------------|
+| threads  | snowflake | an array of [channel](#DOCS_RESOURCES_CHANNEL/channel-object) objects.                                                       |
+| members  | integer   | an array of [thread members](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects, for any returned threads you have joined |
+| has_more | boolean   | whether there are potentially additional threads that could be returned on a subsequent call                                 |
```

Comment:
This is repeated three times, maybe make a referenced object definition.
Also the types for threads and members seem to be inconsistent with the rest of the docs (and just incorrect?) e.g.  the threads property on the guild object seems to be consistent.

Reply:
Oh yeah, those types are very wrong, whoops